src/main/java/com/hedvig/backoffice/graphql/resolvers/GraphQLQuery.java

```diff
@@ -169,4 +170,11 @@ public String me(DataFetchingEnvironment env) {
       .map(SwitchableSwitcherEmail::from)
       .collect(Collectors.toList());
   }
+
+  public List<VoucherCampaign> findPartnerCampaigns() {
+    return productPricingService.searchPartnerCampaigns().stream().map( partnerCampaignSearchResponse ->
+      VoucherCampaign.Companion.from(partnerCampaignSearchResponse)
+    ).collect(Collectors.toList());
+  }
```

Comment:
i have an objection with the naming here. I dont know why, but it feels more clear to me if its getCampaigns instead of search.

feel free to discard though

```suggestion
  public List<VoucherCampaign> getPartnerCampaigns() {
    return productPricingService.getPartnerCampaigns().stream().map( partnerCampaignSearchResponse ->
      VoucherCampaign.Companion.from(partnerCampaignSearchResponse)
    ).collect(Collectors.toList());
  }
```